extensions/vscode/tsconfig.json

```diff
@@ -18,5 +18,5 @@
       "src/*": ["./src/*"]
     }
   },
-  "exclude": ["node_modules", "webviews"]
+  "exclude": ["node_modules", "webviews", "**/**.test.ts", "vitest.config.ts"]
```

Comment:
Ah that's right, test files won't enforce TS rules 🤔 . I'll try to figure out something, because `tsc` complains about them. When included, resolution need to be set as `"Node16"`, but some dependencies for the extension require resolution as it is now `"commonjs"`.

Reply:
I don't mind adding an issue to the backlog / triage board as a follow-up so we can get this in!

Reply:
Oh you fixed it already! Nice!